connectors/sources/sharepoint_online.py

```diff
@@ -682,7 +682,7 @@ class SharepointOnlineAdvancedRulesValidator(AdvancedRulesValidator):
     SCHEMA_DEFINITION = {
         "type": "object",
         "properties": {
-            "maxDataAge": {"type": "integer"},
+            "dontSubextractDriveItemsOlderThan": {"type": "integer"},  # in Days
```

Comment:
`Days` or `seconds`?

https://github.com/elastic/connectors-python/blob/d7172db01603c068e866c08a50ec7e84d403eeb3/connectors/sources/sharepoint_online.py#L1257-L1259

Reply:
You are right, not sure why seconds are there :/